site/en/blog/what-is-the-top-layer/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: "layouts/blog-post.njk"
+title: "What's a \"Top layer\"?"
+description: "With new native components and APIs coming to the platform, what is this \"Top layer\" that they enable us to use?"
```

Comment:
Please don't use native as per style guide. I'd suggest rewording as, "This post explains the "Top Layer" used in some new components and APIs."

Reply:
Yeah, I missed this in my edit, should say "built-in".